scss/utilities/_text.scss

```diff
@@ -12,17 +12,11 @@
 
 @each $breakpoint in map-keys($grid-breakpoints) {
   @include media-breakpoint-up($breakpoint) {
-    $min: breakpoint-min($breakpoint, $grid-breakpoints);
-
-    @if $min {
-      .text-#{$breakpoint}-left   { text-align: left !important; }
-      .text-#{$breakpoint}-right  { text-align: right !important; }
-      .text-#{$breakpoint}-center { text-align: center !important; }
-    } @else {
-      .text-left   { text-align: left !important; }
-      .text-right  { text-align: right !important; }
-      .text-center { text-align: center !important; }
-    }
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
+
+    .text#{$infix}-left   { text-align: left !important; }
+    .text#{$infix}-right  { text-align: right !important; }
+    .text#{$infix}-center { text-align: center !important; }
   }
 }
 
```

Comment:
I didn't remove any of the utilities, `.text-left`, `.text-right` and `.text-center` still exist. Maybe you are a bit confused because of the `#{$infix}`.